unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/NavMap/NavmapView.cs

```diff
@@ -7,32 +7,120 @@ namespace DCL
 {
     public class NavmapView : MonoBehaviour
     {
+        const int LEFT_BORDER_PARCELS = 25;
+        const int BOTTOM_BORDER_PARCELS = 25;
+        const int WORLDMAP_WIDTH_IN_PARCELS = 300;
+
+        [Header("Configuration")]
+        [SerializeField] float parcelHightlightScale = 1.25f;
+
+        [Header("References")]
         [SerializeField] InputAction_Trigger toggleNavMapAction;
         [SerializeField] Button closeButton;
         [SerializeField] ScrollRect scrollRect;
         [SerializeField] Transform scrollRectContentTransform;
         [SerializeField] TextMeshProUGUI currentSceneNameText;
         [SerializeField] TextMeshProUGUI currentSceneCoordsText;
+        RawImage parcelHighlightImage;
+        TextMeshProUGUI highlightedParcelText;
         InputAction_Trigger.Triggered toggleNavMapDelegate;
-
+        int parcelSizeInMap;
         RectTransform minimapViewport;
         Transform mapRendererMinimapParent;
         Vector3 atlasOriginalPosition;
+        MinimapMetadata mapMetadata;
+        Vector3 worldmapOffset;
+        Vector3 worldCoordsOriginInMap;
+        Vector3[] navmapWorldspaceCorners = new Vector3[4];
+        bool cursorLockedBeforeOpening = true;
+        Vector3 mouseMapCoords;
+
+        public bool isOpen
+        {
+            private set
+            {
+                scrollRect.gameObject.SetActive(value);
+            }
+            get
+            {
+                return scrollRect.gameObject.activeSelf;
+            }
+        }
 
         // TODO: Remove this bool once we finish the feature
-        bool enabledInProduction = false;
+        bool enabledInProduction = true;
```

Comment:
If we are going to enable the feature for good, maybe we can remove this bool (disable it otherwise)